src/main/kotlin/org/dashevo/dapiclient/grpc/PlatformMethods.kt

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2020-present, Dash Core Group
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+package org.dashevo.dapiclient.grpc
+
+import com.google.protobuf.ByteString
+import org.dash.platform.dapi.v0.CoreOuterClass
```

Comment:
This import can be removed.

Reply:
Yes.

Reply:
https://github.com/dashevo/dapi-client-android/pull/7/commits/bb430ff16af6894786952476063e82c3125b9d3d